colorcoder/ColorCoder.java

```diff
@@ -0,0 +1,45 @@
+package colorcoder;
+
+/**
+ * Class for color coding functionalities
+ *
+ * @author PDH2COB
+ */
+public class ColorCoder {

```

Comment:
package name and Class name is same....

what does class name "ColorCoder" signify?